modules/gdnative/videodecoder/video_stream_gdnative.cpp

```diff
@@ -355,7 +355,6 @@ RES ResourceFormatLoaderVideoStreamGDNative::load(const String &p_path, const St
 		if (r_error) {
 			*r_error = ERR_CANT_OPEN;
 		}
-		memdelete(f);
```

Comment:
This is fine, but you do need to `memdelete(f)` when it is actually valid.

Reply:
Made the change.